docs/rust.md

```diff
@@ -1,3 +1,33 @@
 # Awesome Open Source - Rust
 
-Coming Soon!
+- [actix-web](https://github.com/actix/actix-web) - Actix Web is a powerful, pragmatic, and extremely fast web framework for Rust.
+- [alacritty](https://github.com/alacritty/alacritty) — A cross-platform, GPU enhanced terminal emulator
+- [Asymmetrik/ngx-deploy-azure](https://github.com/Asymmetrik/ngx-deploy-azure) - Deploy Angular apps to Azure.
```

Comment:
this repo doesn't exist

Reply:
Removed. Added 2 more projects in favor of removals.